terraform/modules/application/alarms.tf

```diff
@@ -278,7 +298,7 @@ resource "aws_cloudwatch_metric_alarm" "healthy_hosts_count" {
   ok_actions                = var.suppress_ok_notifications ? [] : [var.sns_topic_arn]
   period                    = 60
   statistic                 = "Minimum"
-  threshold                 = var.app_task_desired_count
+  threshold                 = local.app_desired_count_for_alarms
```

Comment:
This does make this alarm less stringent - given that we almost inevitably have the unhealthy hosts count alarm as well, I think we're ok with that.